### PR TITLE
Update loss calculation in quickstart documentation

### PR DESCRIPTION
**Pull request by francislata**
# Overview
This pull request updates `quickstart.md` for the tinygrad MNIST example when computing the loss. When looking at how it is implemented now, it is directly part of `Tensor` and not as a standalone function. So I'm updating this documentation along the way.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ with Tensor.train():
     out = net(batch)
 
     # compute loss
-    loss = sparse_categorical_crossentropy(out, labels)
+    loss = out.sparse_categorical_crossentropy(labels)
```
Comment (wozeparrot) [inline docs/quickstart.md:191]: It's defined on line 138. This was to show the implementation that is defined above.

Comment (francislata) [reply docs/quickstart.md:191]: Ah, got it. I definitely skipped that codeblock when I started to write the training code portion.
